### PR TITLE
test of manual nav

### PR DESCRIPTION
**Pull request by billmcmillin**

### CODE REVIEW

**Comment by kateefly** [comment]
This looks good so far! A couple notes:

- A few pages in the Sandbox are missing the /entries prefix they need (Glossary, Collaborators, Contributors) since they are in the entries folder

- You can comment out the link to Glossary & Resources since we're not quite ready to have that page go live yet

- [Tools](https://github.com/DLFMetadataAssessment/Tools) & the [Map Clearinghouse](https://github.com/DLFMetadataAssessment/MetadataSpecsClearinghouse) have their own child repositories that can be linked to

**Comment by billmcmillin** [comment]
Had to use absolute URLs because navigation across repos isn't working.

**Comment by kateefly** [comment]
Looks good except for one minor point - the links to the Framework pages aren't individual entries but are anchor tags on a page - so /Framework/entries/41framework.html#Metadata_metrics , etc.

**Comment by kateefly** [comment]
Here's a cheat sheet here if you need it: https://github.com/DLFMetadataAssessment/Framework/blob/master/_includes/nav.html

File: _includes/nav.html [hunk 1]
```diff
@@ -1,27 +1,33 @@
 <ul id="nav">
-  <li><a href="/">DLF AIG Metadata Working Group Toolkit</a></li>
-  <li><a href="/about.html">About &amp; Contact</a></li>
-  <li><a href="/take-part.html">Participate</a></li>
-  <li><a href="/why.html">Importance of Metadata Assessment</a></li>
-  <li><a href="/review.html">Environmental Scan</a>
+  <li><a href="https://dlfmetadataassessment.github.io/Tools">DLF AIG Metadata Working Group Toolkit</a></li>
```
Comment (kateefly) [inline _includes/nav.html]: Sandbox main page (https://dlfmetadataassessment.github.io/Sandbox)

File: _includes/nav.html [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
+		</ul>
   </li>
-  <li><a href="/tools.html">Tools for Metadata Assessment</a></li>
-  <li><a href="/profiles.html">Metadata Profiles &amp; Mappings Clearinghouse</a></li>
-  <li><a href="/resources.html">Glossary &amp; Resources</a></li>
-  <li><a href="/collaborations.html">Collaborations</a></li>
-  <li><a href="/contributors.html">Contributors</a></li>
-</ul>
\ No newline at end of file
+  <li><a href="https://dlfmetadataassessment.github.io/EnvironmentalScan/entries/34tools">Tools for Metadata Assessment</a></li>
```
Comment (kateefly) [inline _includes/nav.html]: http://dlfmetadataassessment.github.io/Tools

**Comment by kateefly** [comment]
De-expand Framework. Will have to update nav again when live to production.

**Pull request merged**